pages/glossary/disabled-element.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Disabled Element
+key: disabled-element
+---
+
+An element that is normally [perceivable][] and [operable][], but has been rendered [inoperable][operable] either temporarily or permanently using one of the following properties, is considered disabled:
```

Comment:
I can't remember what rule it was, but we recently revised one where the WCAG definition for operable was considered ambiguous, and thus could not be used in the applicability. This also needs to be addressed here. Possibly also for perceivable.

Also, I would say that normally might be ambiguous and is certainly subjective.